db/migrate/20191031131016_add_created_at_index_to_users.rb

```diff
@@ -1,4 +1,6 @@
 class AddCreatedAtIndexToUsers < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
```

Comment:
How was this able to pass the specs without this? I would have assumed this would have failed when we migrated the test db

Reply:
The specs don't run the migrations, they load the `schema.rb`, and this statement tells Rails to disable transactions, it doesn't leave a trace in the schema file.

I usually test branches I'm reviewing but I didn't get around to that in time :(

Reply:
Same, I take that for granted that not everyone might do that. We should probably run the migrations or any new migrations in our spec process

Reply:
yeah, let's open an issue about this, lest we forget